Resources/Changelog/GoobChangelog.yml

```
@@ -11325,3 +11343,41 @@ Entries:
       message: Traitors should get their uplinks now
   id: 1245
   time: '2025-06-16T13:16:56.0000000+00:00'
+- author: Aidenkrz
+  changes:
+    - type: Add
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)